pages/api/logs/index.ts

```diff
@@ -0,0 +1,51 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { ApiError, LogRequest } from 'utils/types'
+import { serverAuth } from 'utils/auth'
+import * as MongoDriver from 'server/actions/MongoDriver'
+import LogSchema from 'server/models/Log'
+import { apiLogValidation } from 'utils/apiValidators'
+import { getLogs } from 'server/actions/Logs'
+
+// @route GET api/logs - Returns a list of all logs in the database - Private
+// @route POST /api/logs - Create a logs from request body - Private
+export default async function LogsHandler(
```

Comment:
```suggestion
export default async function logsHandler(
```
to keep consistent with other handlers

Reply:
I keep missing this! I guess I'm used to the C# pattern... good catch